src/components/dropdowns/dropdown-complex-search/DropdownComplexSearch.js

```diff
@@ -0,0 +1,150 @@
+    showOptions
+      ? `2px solid ${colors.emerald400}`
+      : `2px solid ${colors.grey500}`};
+  color: ${({ selectedOption }) =>
+    selectedOption ? `${colors.emerald400}` : `${colors.grey500}`};
+
+  &:hover {
+	color: ${({ error }) => (error ? `white` : ` ${colors.emerald400}`)}
+    border: 2px solid ${colors.emerald400};
+  }
+
+  border: ${({ error }) => (error ? `2px solid ${colors.red500}` : '')};
+`;
+
+const OptionsWrapper = styled.div`
+  ${OptionsWrapperCSS}
+`;
+
+const Option = styled.div`
+  ${OptionCSS}
+  width: 320px;
+  border-radius: ${({ isLastOption }) => (isLastOption ? '0 0 6px 6px' : '')};
+  margin-bottom: ${({ isLastOption }) => (isLastOption ? '-1px' : '')};
+`;
+
+const ErrorMsg = styled.div`
+  ${ErrorMsgCSS}
+`;
+
+const DropdownComplexSearchComponent = ({ placeholder, options, error }) => {
+  const [showOptions, setShowOptions] = useState(false);
+  const [selectedOption, setSelectedOption] = useState('');
+  const [hasError, setHasError] = useState(error);
+
+  const displayPrompt = (value) => {
+    if (!value) {
+      // Render "Start typing to search"
+      return (
+        <OptionsWrapper>
+          <Option>Start typing to search...</Option>
+        </OptionsWrapper>
+      )
+    }
+
+    else {
+      // Loop through options to filter schools when typing
+      const filteredSchools = []
+      for (let i=0; i < options.length; i++) {
+        // If found school, append to filteredSchools
+        if (options[i].includes(value)) {
+          filteredSchools.push(options[i])
+        }
+      }
+
+      if (filteredSchools.length > 0) {
+        // Render items in filteredSchools as options
+        return (
+          <OptionsWrapper>
+            {filteredSchools.map((option, index) => {
+              return (
+                <Option
+                  key={index}
+                  onClick={() => {
+                    setSelectedOption(option);
+                    setShowOptions(false);
+                    setHasError(false);
+                  }}
+                  isLastOption={index === filteredSchools.length - 1}
+                >
+                  {option}
+                </Option>
+              );
+            })}
+          </OptionsWrapper>
+        )
+      }
+      else { 
+        // Render "My school is not listed, use "event.target.value"
+        return (
+          <OptionsWrapper>
+            <Option
+              onClick={() => {
+                setSelectedOption(value);
+                setShowOptions(false);
+                setHasError(false);
+              }}>
+                My school is not listed, use {value}
+            </Option>
+          </OptionsWrapper>
+        )
+      }
+    }
+  };
+
+  return (
+    <Wrapper>
+      <Image src={Search} alt='Search icon' />
+
+      <Input
+        placeholder={placeholder}
+        showOptions={showOptions}
+        selectedOption={selectedOption}
+        onInput={() => setShowOptions(!showOptions)}
```

Comment:
The prompt only shows half the time when entering an input. Setting this to `onInput={() => setShowOptions(true)}` will fix the problem